### PR TITLE
Upload draft blog post to recap UGM and 2024 workshops

### PR DESCRIPTION
**Pull request by jennaswa**

### CODE REVIEW

**Review by orbeckst — APPROVED** [review]
Great, thank you so much for putting it all together @jennaswa !

File: _posts/2024-09-11-mda_2024events.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+## MDAnalysis Training Workshops
+
+This year was an exciting year for MDAnalysis, in that we were able to offer 4 different *free* online and hybrid training workshops!
+
+### Februrary 28th: Introduction to MDAnalysis and Molecular Nodes (Online)
+
+Around 40 people joined us live on February 28, 2024 for an online workshop, during which 4 instructors (@fiona-naughton, @lilyminium, @yuxuanzhuang, @BradyAJohnston) introduced the MDAnalysis package and provided an interactive tutorial for visualizing MDAnalysis data in [Molecular Nodes](https://bradyajohnston.github.io/MolecularNodes/); the [recording](https://youtu.be/3zKBjnRnAMg?si=cnGSQxyUmzc9zCDd) is available on our [YouTube](https://www.youtube.com/@mdanalysis3040) channel. All workshop materials and installation instructions are publicly available on the MDAnalysis/MDAnalysisWorkshop2-Intro0.5Day GitHub repository on the ['feb24-ws' branch](https://github.com/MDAnalysis/MDAnalysisWorkshop-Intro0.5Day/tree/feb24-ws).
```
[the remark this answers is not in the document]
Comment (RMeli) [reply _posts/2024-09-11-mda_2024events.md:81]: The same comment applies to the branches below.

File: _posts/2024-09-11-mda_2024events.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+![Micaela Matta Leading the In-Person Instruction for the MDAnalysis Workshop at UCL](/public/images/May2024_Workshop_Photo.jpg)
+
+### June 24th-25th: Moving from User to Developer: Analyzing Molecular Simulations and Building New Tools
+
+In late June 2024, an MDAnalysis instructor team (@IAlibay, @orbeckst, @fiona-naughton, @yuxuanzhuang, @mikemhenry, @ianmkenney) partnered with the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI) to offer a 2-day hybrid workshop at Arizona State University (ASU) in the [Center for Biological Physics](https://cbp.asu.edu/) covering beginner to intermediate tutorials on the [MDAnalysis library](https://github.com/MDAnalysis/mdanalysis) as well as tutorials on software development best practices provided by MolSSI instructors (@sjayellis, @armcdona). A total of 35 individuals attended in person, with another up to 76 attending online. All workshop materials are now publicly available in the MDAnalysisMolSSIWorkshop-Intermediate2Day GitHub repository under the ['jun24-ws' branch](https://github.com/MDAnalysis/MDAnalysisMolSSIWorkshop-Intermediate2Day/tree/jun24-ws).
```
Comment (RMeli) [inline _posts/2024-09-11-mda_2024events.md:91]:
```suggestion
In late June 2024, an MDAnalysis instructor team (@IAlibay, @orbeckst, @fiona-naughton, @yuxuanzhuang, @mikemhenry, @ianmkenney) partnered with the [Molecular Sciences Software Institute](https://molssi.org/) (MolSSI) to offer a 2-day hybrid workshop at [Arizona State University (ASU)](https://www.asu.edu) in the [Center for Biological Physics](https://cbp.asu.edu/) covering beginner to intermediate tutorials on the [MDAnalysis library](https://github.com/MDAnalysis/mdanalysis) as well as tutorials on software development best practices provided by MolSSI instructors (@sjayellis, @armcdona). A total of 35 individuals attended in person, with another up to 76 attending online. All workshop materials are now publicly available in the MDAnalysisMolSSIWorkshop-Intermediate2Day GitHub repository under the ['jun24-ws' branch](https://github.com/MDAnalysis/MDAnalysisMolSSIWorkshop-Intermediate2Day/tree/jun24-ws).
```

File: _posts/2024-09-11-mda_2024events.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jennaswa) [reply _posts/2024-09-11-mda_2024events.md:81]: Yes, there was some discussion started about this on Discord earlier this year in January: https://discord.com/channels/807348386012987462/817022693676285992/1201343337878265887. I'll also add it to the [next business meeting agenda](https://docs.google.com/document/d/1Bz_KnH9TfkuNozBG0j0urK8H9WhD_uAQobeLBbrBbYg/edit#heading=h.2sfic1bxtuk5), though there might not be time to get to it, so we don't lose track of trying to reignite the conversation and making a longer term plan.

**Pull request merged**